resizer.go

```diff
@@ -125,18 +125,22 @@ func resizer(buf []byte, o Options) ([]byte, error) {
 }
 
 func loadImage(buf []byte) (*C.VipsImage, ImageType, error) {
+	return loadImageWithOptions(buf, Options{})
```

Comment:
Users who expect `Convert(...)` to convert the _last_ page of multi-page images will notice a change here, as this will default it to convert the _first_ page. 

Converting the first — rather than the last — page seems like a sensible default, but I don't want to break expected behavior either. 

`Options` could be initialized with a sentinel value  like `-1` here to revert this to the previous behavior. But, I'm hesitant to do that because the previous behavior is probably not what many users expect.

Reply:
Converting the first page by default [seems to be the previous behavior](https://stackoverflow.com/questions/47492837/golang-convert-pdf-to-image-by-bimg) - would it really be a breaking change?

Reply:
Breaking changes aren't bad, we just need to make sure it's properly conveyed when releasing. Perhaps a minor version bump with changes like this?